src/main/java/com/epam/aidial/cfg/domain/model/source/InterceptorRunnerSource.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.aidial.cfg.domain.model.source;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class InterceptorRunnerSource extends InterceptorSource {
+    private String templateName;
```

Comment:
let's try to use either interceptor`Runner` or interceptor`Template` in all places.